src/Stackage/Snapshot/Diff.hs

```diff
@@ -77,7 +77,7 @@ snapshotDiff
 snapshotDiff as0 bs0 =
     SnapshotDiff $ map (second VersionChange) $ go (sortEm as0) (sortEm bs0)
   where
-    sortEm = sortOn (toCaseFold . textDisplay . fst)
+    sortEm = sortOn (textDisplay . fst)
```

Comment:
This is still not obviously correct, as we sort on `textDisplay` but then compare later without it.
Is`textDisplay` necessary here?

NB: Will the order change by removing `toCaseFold` so that packages with capital letters come first, and then the lowercase ones?  This might not be desirable...

Reply:
Thank you, good point

Reply:
I took your approach